SPO/AppInsightsExtensionSolutionSample-Snippet/src/extensions/appInsights/AppInsightsApplicationCustomizer.ts

```diff
@@ -0,0 +1,47 @@
+    src: "https://az416426.vo.msecnd.net/scripts/b/ai.2.gbl.min.js", // The SDK URL Source
+    //name: "appInsights", // Global SDK Instance name defaults to "appInsights" when not supplied
```

Comment:
You could remove these commented out entries and perhaps include a comment which links to the main GitHub page that explains the values and that the snippet can be copied (when updated from).
Current snippet: https://github.com/Microsoft/ApplicationInsights-JS#snippet-setup-ignore-if-using-npm-setup
Options: https://github.com/Microsoft/ApplicationInsights-JS#snippet-configuration-options

You can "detect" what the current version is by looking for "sv:" in the snippet, both are current version 4 from the above snippet ",queue:[],sv:"4",version:2,")